src/main/webapp/portal/main/partials/access-denied.html

```diff
@@ -0,0 +1,13 @@
+<frame-page app-header-title='Access denied' 
```

Comment:
Is there an opportunity to confirm to the user what application it is that they're being denied access to, so they can try to understand whether they went somewhere they didn't mean or they are being denied access to an application they felt meant to access?

>  [If it’s an application, include the header to provide context.](http://staging-2.uc.wisc.edu/my-uw/#coarse-grain-access)